app/src/main/java/fr/gaulupeau/apps/Poche/data/Settings.java

```diff
@@ -479,6 +479,14 @@ public void setDisableTouchKeyCode(int keyCode) {
         setInt(R.string.pref_key_ui_disableTouch_keyCode, keyCode);
     }
 
+    public int getScrolledOverBottom() {
+        return getInt(R.string.pref_key_ui_scrollOverBottom, 3);
```

Comment:
what is the hardcoded three (3) about?

Reply:
That's the initial default value, so without changing the setting, you would need to scroll 3 times over the bottom to mark as read. Is there a different location where such default values should be stored?

Reply:
good question. I do  not know. 
Isn't there a relation to this line https://github.com/wallabag/android-app/pull/648/files#diff-07a6745e4ce5bc268d522049aaabf1b3R158 ?

Reply:
Quite possibly, thanks, I'll check and test this (and the other PR) on the weekend.

Reply:
Thanks a lot for your ongoing contribution. I just had a look into the source and i saw that the location for the default value is okay like that. It is just confusing with the two different default values.

Reply:
I can help here, so that we can merge this PR before the weekend. What should be the default value? 3 or 10?

Reply:
That would be great!
Default should be 3 (imagine 10 times pushing the button for an effect :P ). Otherwise, you could also try yourself and see what feels good?